packages/sdk/src/across/gasFeeCalculator.ts

```diff
@@ -38,21 +35,70 @@ export default async (
     gasPrice = await ethersProvider.getGasPrice();
   }
 
-  if (tokenAddress === undefined) {
-    const gasFees = calculateGasFees(gas, gasPrice);
-    return {
-      gasFees,
-      feesAsPercent: percent(gasFees, totalRelayed).toString(),
-    };
+  // this means the token is eth, address undefined or address zero
+  if (tokenAddress === constants.ADDRESSES.ETH) {
+    return calculateGasFees(gas, gasPrice);
   }
   const coingecko = new Coingecko();
   const [, tokenPrice] = await coingecko.getCurrentPriceByContract(tokenAddress, "eth");
   const erc20Client = erc20Connect(tokenAddress, ethersProvider);
   const decimals = await erc20Client.decimals();
-  const gasFees = calculateGasFees(gas, gasPrice, tokenPrice, decimals);
+  return calculateGasFees(gas, gasPrice, tokenPrice, decimals);
+}
 
+type GasTable = Record<string, number> & { DEFAULT: number };
+// These tables are hard coded to mainnet
+function makeSlowGasTable(): GasTable {
+  return {
+    [constants.ADDRESSES.UMA]: constants.SLOW_UMA_GAS,
+    [constants.ADDRESSES.ETH]: constants.SLOW_ETH_GAS,
+    [constants.ADDRESSES.WETH]: constants.SLOW_ERC_GAS,
+    DEFAULT: constants.SLOW_ERC_GAS,
+  };
+}
+// instant gas is the amount of gas above slow gas
+function makeInstantGasTable(): GasTable {
   return {
-    gasFees,
-    feesAsPercent: percent(gasFees, totalRelayed).toString(),
+    [constants.ADDRESSES.UMA]: constants.FAST_UMA_GAS - constants.SLOW_UMA_GAS,
+    [constants.ADDRESSES.ETH]: constants.FAST_ETH_GAS - constants.SLOW_ETH_GAS,
+    [constants.ADDRESSES.WETH]: constants.FAST_ERC_GAS - constants.SLOW_ERC_GAS,
+    DEFAULT: constants.FAST_ERC_GAS - constants.SLOW_ERC_GAS,
   };
+}
+
+const GetGasByAddress = (gasTable: GasTable) => (tokenAddress: string): number => {
+  if (exists(gasTable[tokenAddress])) return gasTable[tokenAddress];
+  return gasTable.DEFAULT;
+};
+
+const getInstantGasByAddress = GetGasByAddress(makeInstantGasTable());
+const getSlowGasByAddress = GetGasByAddress(makeSlowGasTable());
+
+type ReturnGetDepositFees = {
+  slowPct: string;
+  instantPct: string;
 };
+
+/**
+ * getDepositFees. Returns to you appropriate values for calling the relay deposit function. Returns the slow gas fee
+ * and the instant gas fee which is the amount additional to the slow fee as percentages in wei.
+ *
+ * @param {Provider} ethersProvider - Read provider on mainnet
+ * @param {BigNumberish} amountToRelay - Amount in wei of token to relay
+ * @param {string} tokenAddress - Mainnet address of token to relay, for ETH specify constants.ADDRESSES.ETH
+ * @returns {Promise<ReturnGetDepositFees>} - Returns the fee parameters to the deposit function on the deposit box contract. These are percentages in wei.
```

Comment:
nit: might be unclear what a percentage is in wei (for people outside of our org). Could we add an example? `ex. 50% is represented as 0.5 * 1e18`.